addons/mrp/wizard/mrp_product_produce.py

```diff
@@ -55,6 +55,7 @@ def default_get(self, fields):
         'finished_product_produce_id', string='By-products')
     production_id = fields.Many2one('mrp.production', 'Manufacturing Order',
         required=True, ondelete='cascade')
+    qty_remaining = fields.Float('Quantity To Be Produced', compute='_get_todo', digits='Product Unit of Measure')
```

Comment:
use the _compute_pending_production or modify _get_todo to _compute_get_todo and use this field instead of call to function